src/main/webapp/i18n/de/quizExercise.json

```
@@ -261,6 +262,12 @@
                 "individual": "Einzeln",
                 "explanation": "Ausführungsmodus bestimmt wie Studierende an einem Quiz teilnehmen können.\n\nSynchronisiert: Alle Studierenden nehmen gleichzeitig an dem Quiz teil.\nGruppiert: Es gibt mehrere Gruppen an denen Studierende mit einem Passwort teilnehmen können.\nEinzeln: Studierende können jederzeit selbstständig an dem Quiz teilnehmen."
             },
+            "releaseDateExplanation": {
+                "synchronized": "Synchronisiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst ab dem Startdatum sehen.",
+                "batched": "Gruppiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst sehen, wenn ein Stapel gestartet wird und sie das Passwort dafür eingeben.",
```

Same, tho i am open for better translations for this, both in german and english, it should at least be consistent.
```suggestion
                "batched": "Gruppiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst sehen, wenn eine Gruppe gestartet wird und sie das Passwort dafür eingeben.",
```